pkg/grpc/auditlogstore/audit_log_store_test.go

```diff
@@ -2,6 +2,7 @@ package auditlogstore
 
 import (
 	"bytes"
+	"context"
```

Comment:
i think this one will be fine though! because `context.Background()` satisfies the `Context` interface from either package. confusing stuff